app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str):
+    old_file = command.split()[1]
+    new_file = command.split()[2]
+    if old_file != new_file:
```

Comment:
your should also check if command.split(" ")[0] == "cp". Refactor this block to leave from function if source_filename is equal to destination_filename or command.split(" ")[0] != "cp"

Reply:
The tests have passed, so everything is OK! Many checks can be written: Is there a "command" in the string? does the string consist of 3 parameters separated by a space? Or maybe parameter 3 contains incorrect characters in the name? and others. The task indicated writing a working algorithm, and not implementing all possible exceptions

Reply:
Ok, you are welcome

Reply:
You definitely need to check if the command is correct